SmartCar.ino

```diff
@@ -1,4 +1,5 @@
 #include <Smartcar.h>
+#include <Wire.h>

```

Comment:
You shouldn't need to include `Wire.h` when compiling with the Arduino IDE :cactus: